with-antd/theme.tsx

```diff
@@ -8,6 +9,8 @@ export const ThemeProvider = ({ children = null as ReactNode }) => (
         colorPrimary: "#a1701d"
       }
     }}>
-    {children}
+      <StyleProvider container={document.querySelector("plasmo-csui").shadowRoot}>
```

Comment:
We might want to move this provider out into the CSUI itself IMO :-?.... OR preferably using getStyle so that we can isolate CSUI from each other.

Reply:
I will go back to seasoning this next Monday then merge it to the upcoming plasmo patch!

Reply:
ok, thank you :D